src/ivoc/htlist.h

```diff
@@ -66,21 +55,9 @@ inline bool HTList::IsEmpty() {
 inline HTList* HTList::First() {
     return _next;
 }
-inline HTList* HTList::Last() {
```

Comment:
if there's a first there should be a last. it might not exist out of lazyness, but removing it feels excessive.

Reply:
the purpose of this patch was to reduce this class as most as possible, as no one will use it anymore (I hope). So when trying to replace by something else this become easier.